src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star_stars"></div>
+      <div class="star_stars"></div>
+      <div class="star_stars"></div>
+      <div class="star_stars"></div>
+      <div class="star_stars"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
```
classes in html mismatch with styles. and follow bem convention and bem linter for naming bem elements